src/System.Private.CoreLib/shared/System/Diagnostics/Debug.cs

```diff
@@ -347,4 +351,160 @@ private sealed class DebugAssertException : Exception
 
         internal static Action<string> s_WriteCore = WriteCore;
     }
+
+    internal class DebugInternal : TraceDebugBase
```

Comment:
Note: Debug class and its methods have always been static. DebugInternal class (used by Debug) is no longer static since we are using base class.